charts/external-dns/templates/deployment.yaml

```
@@ -44,6 +44,9 @@ spec:
       {{- with .Values.terminationGracePeriodSeconds }}
       terminationGracePeriodSeconds: {{ . }}
       {{- end }}
+      {{- if .Values.dnsPolicy }}
+      dnsPolicy: {{ .Values.dnsPolicy }}
+      {{- end }}
```

```suggestion
      {{- with .Values.dnsPolicy }}
      dnsPolicy: {{ . }}
      {{- end }}
```

We prefer `with` over `if` for simple cases such as this.